fdbserver/ptxn/Serializer.h

```diff
@@ -98,20 +92,127 @@ class HeaderedItemsSerializerBase {
 		    0, reinterpret_cast<uint8_t*>(const_cast<void*>(writer.getData())) + getHeaderBytes(), getItemsBytes());
 	}
 
+	// Mark the serializer not accepting more items, and the header is ready for writing
```

Comment:
See https://google.github.io/styleguide/cppguide.html, "Function Comments" section. The verb needs an "s".
```suggestion
	// Marks the serializer not accepting more items, and the header is ready for writing
```